src/pycountry/__init__.py

```diff
@@ -113,7 +115,7 @@ def add_result(country: "pycountry.db.Country", points: int) -> None:
             raise LookupError(query)
 
         sorted_results = [
-            self.get(alpha_2=x[0])
+            cast(pycountry.db.Country, self.get(alpha_2=x[0]))
```

Comment:
From looking at the code this should not be `Optional` since how the keys are found, but the `self.get(...)` call returns an optional value, so cast this to non optional.